helix-cli 4.0.2 and newer missing @adobe/htlengine in deployed runtime actions

**Description**
The greenkeeper PRs to update helix-cli for the devsite have been failing since 4.0.2. See the following failed CI jobs:

- 4.0.2: https://circleci.com/gh/adobe/developer.adobe.com/2720?utm_campaign=chatroom-integration&utm_medium=referral&utm_source=slack
- 4.0.3: https://circleci.com/gh/adobe/developer.adobe.com/2727?utm_campaign=chatroom-integration&utm_medium=referral&utm_source=slack
- 4.0.4: https://circleci.com/gh/adobe/developer.adobe.com/2734?utm_campaign=chatroom-integration&utm_medium=referral&utm_source=slack

**To Reproduce**
Steps to reproduce the behavior:
1. Visit https://adobedevsite.helix-demo.xyz/xd/docs
2. See the error which looks like:

```
{
  "code": "e79xNnpAi1u7m37FWow781fgoK5LJ60c",
  "error": "There was an error processing your request."
}
```

3. The same error can be seen when issuing a request against runtime directly: https://adobeioruntime.net/api/v1/web/developer-adobe-com/github-com--adobe--developer-adobe-com--greenkeeper--adobe-helix-cli-4-0-4-dirty/html?owner=AdobeXD&repo=plugin-docs&ref=master&path=/README.md&selector=&extension=&strain=xd-docs-dev&rootPath=/xd/docs&params=

**Expected behavior**
No runtime action explosions.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Version:**
4.0.2, 4.0.3, 4.0.4

**Additional context**
Runtime activation logs has this interesting tidbit:

```
    "response": {
        "status": "action developer error",
        "statusCode": 0,
        "success": false,
        "result": {
            "error": "Initialization has failed due to: Error: Cannot find module '@adobe/htlengine'\n    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)\n    at Function.Module._load (internal/modules/cjs/loader.js:507:25)\n    at Module.require (internal/modules/cjs/loader.js:637:17)\n    at require (internal/modules/cjs/helpers.js:22:18)\n    at Object.<anonymous> (/nodejsAction/XXHKndDO/html.js:30:5)\n    at Module._compile (internal/modules/cjs/loader.js:689:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)\n    at Module.load (internal/modules/cjs/loader.js:599:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:530:3)"
```

--- a/package.json
+++ b/package.json
@@ -109,7 +109,8 @@
     "proxyquire": "^2.1.0",
     "semantic-release": "^15.13.12",
     "sinon": "7.3.2",
-    "unzip2": "0.2.5"
+    "unzip2": "0.2.5",
+    "xregexp": "^4.2.4"
   },
   "snyk": true,
   "lint-staged": {

--- a/src/package.cmd.js
+++ b/src/package.cmd.js
@@ -135,13 +135,19 @@ class PackageCommand extends StaticCommand {
         // if the module was linked via `npm link`, then it is a checked-out module, and should
         // not be included as-is.
         const modPath = info.externals[mod];
-        if (modPath.indexOf('/node_modules/') < 0 || modPath.indexOf('\\node_modules\\') < 0) {
+        if (modPath.indexOf('/node_modules/') < 0 && modPath.indexOf('\\node_modules\\') < 0) {
           // todo: async
           // todo: read .npmignore
-          const files = glob.sync('!(.git|node_modules|logs|docs|coverage)/**', {
-            cwd: modPath,
-            matchBase: false,
-          });
+          const files = [
+            ...glob.sync('!(.git|node_modules|logs|docs|coverage)/**', {
+              cwd: modPath,
+              matchBase: false,
+            }),
+            ...glob.sync('*', {
+              cwd: modPath,
+              matchBase: false,
+              nodir: true,
+            })];
           files.forEach((name) => {
             archive.file(path.resolve(modPath, name), { name: `node_modules/${mod}/${name}` });
           });

--- a/test/testPackageCmd.js
+++ b/test/testPackageCmd.js
@@ -70,7 +70,7 @@ describe('hlx package (Integration)', () => {
 
     await assertZipEntries(
       path.resolve(buildDir, 'html.zip'),
-      ['package.json', 'html.js', 'html.pre.js', 'helper.js', 'third_helper.js'],
+      ['package.json', 'html.js', 'html.pre.js', 'helper.js', 'third_helper.js', 'node_modules/@adobe/htlengine/package.json'],
     );
     await assertZipEntries(
       path.resolve(buildDir, 'static.zip'),

